gcn_monitor/kafka.py

```diff
@@ -23,7 +26,30 @@ def stats_cb(data):
         metrics.broker_state.labels(broker["name"]).state(broker["state"])
 
 
-def run():
+def parse_filenames(message):
+    """Parses a Kafka message into a file names for s3 upload.
+
+    Args:
+        message (Message): Any Kafka message
+
+    Returns:
+        fileName (string): The Key for S3's put_object method, formatted as `topics/<topic>/<encodedPartition>/<topic>+<kafkaPartition>+<startOffset>.<format>` defined at https://docs.confluent.io/kafka-connectors/s3-sink/current/overview.html#s3-object-names
+        messageKeyFileName (string): The Key for S3's put_object method, formatted as `topics/<topic>/<encodedPartition>/<topic>+<kafkaPartition>+<startOffset>.<format>` defined at https://docs.confluent.io/kafka-connectors/s3-sink/current/overview.html#s3-object-names
+        headersFileName (string): The Key for S3's put_object method, formatted as `topics/<topic>/<encodedPartition>/<topic>+<kafkaPartition>+<startOffset>.<format>` defined at https://docs.confluent.io/kafka-connectors/s3-sink/current/overview.html#s3-object-names
+
+    """
```

Comment:
Please use [Numpydoc](https://numpydoc.readthedocs.io/en/latest/format.html)-style docstrings. For an example, see https://github.com/nasa-gcn/gcn-classic-to-kafka/blob/main/gcn_classic_to_kafka/common.py#L14-L37.

Also, the text `defined at https://docs.confluent.io/kafka-connectors/s3-sink/current/overview.html#s3-object-names` is repeated three times. Can you provide that reference more succinctly?